static/js/components/NewObservation.jsx

```diff
@@ -34,29 +54,26 @@ const NewObservation = () => {
         title: "Observation file",
         description: "Observation file",
       },
-      instrument_id: {
-        type: "integer",
-        oneOf: instrumentList.map((instrument) => ({
-          enum: [instrument.id],
-          title: `${
-            telescopeList.find(
-              (telescope) => telescope.id === instrument.telescope_id
-            )?.name
-          } / ${instrument.name}`,
-        })),
-        title: "Instrument",
-        default: instrumentList[0]?.id,
-      },
     },
-    required: ["file", "instrument_id"],
+    required: ["file"],
   };
 
+  // Render the observation form with the NewDropDownSearchBar component
   return (
     <Form
       schema={observationFormSchema}
       validator={validator}
       onSubmit={handleSubmit}
-    />
+    >
+      <NewDropDownSearchBar
```

Comment:
Please move that outside of the form (like above it), it does not do anything to embed it in the form. There is actually a proper way to embed components in a rjsf fromSchema, but not worth doing that here. When you move it up, make sure that you also add a little space (where this NewObservation component is called) between the component and the `Add new observations` typography that's above it.

Reply:
I moved SearchableDropdown component outside of the form. Could you clarify how I should add a little space between the NewObservation component and the typography in the image below?

<img width="1840" alt="image" src="https://github.com/skyportal/skyportal/assets/105004472/355b64f4-c4d4-45d2-ab42-16e82b228ee9">